hack/update-bundled-plugin-runtime-version/update-bundled-plugin-runtime-version.txt

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package version
+
+// IMPORTANT: Please !!DO NOT!!! change the content of this file manually
+// This file is auto-generated based on the version details provided during build time
+// and the values mentioned here are used to store the library version
+
+// All the variables are set during build time
+// Note: Please !!DO NOT!!! change the name or remove this variable
+var (
+	Version string = "TANZU_PLUGIN_RUNTIME_VERSION"
+)
```

Comment:
This file seems to need an empty line at the end.
If not, when I do `make configure-bom` the new ` pkg/v1/cli/version/zz_generated_plugin_runtime_version.go` gets modified.

Reply:
Thanks @marckhouzam . Looks like my editor added a space for the `go` file when i saved it. Added the space in this txt file to avoid this.